pkg/webhook/webhook.go

```diff
@@ -15,37 +15,151 @@
 package webhook
 
 import (
+	"encoding/json"
 	"io/ioutil"
 	"net/http"
-
-	"k8s.io/klog/v2"
+	"time"
 
 	"antrea.io/resource-auditing/pkg/gitops"
+	"antrea.io/resource-auditing/pkg/types"
+
+	"github.com/go-git/go-git/v5/plumbing/object"
+	"k8s.io/klog/v2"
 )
 
+type Change struct {
+	Sha     string `json:"sha"`
+	Author  string `json:"author"`
+	Message string `json:"Message"`
+}
+
+type Filters struct {
+	Author    string    `json:"author"`
+	Since     time.Time `json:"since"`
+	Until     time.Time `json:"until"`
+	Resource  string    `json:"resource"`
+	Namespace string    `json:"namespace"`
+	Name      string    `json:"name"`
+}
```

Comment:
do these need to be part of this PR?

Reply:
forgot to remove these, they'll be part of Dhruv's PR for the get changes cli command/handler - thanks for catching this